beacon-chain/blockchain/service.go

```diff
@@ -341,3 +350,76 @@ func (c *ChainService) processBlock(block *types.Block) error {
 
 	return nil
 }
+
+func (c *ChainService) calculateNewBlockVotes(block *types.Block, aState *types.ActiveState, cState *types.CrystallizedState) error {
+	for i := 0; i < len(block.Attestations()); i++ {
```

Comment:
I suppose the following will be easier : )
```
for _, attestation := range block.Attestations() {
  ...
}
```

Reply:
Sure. Thanks!